posthog/tasks/exports/image_exporter.py

```diff
@@ -227,18 +226,13 @@ def _screenshot_asset(
         driver.save_screenshot(image_path)
     except Exception as e:
         # To help with debugging, add a screenshot and any chrome logs
-        with configure_scope() as scope:
-            scope.set_extra("url_to_render", url_to_render)
+        with posthoganalytics.new_context():
+            posthoganalytics.tag("url_to_render", url_to_render)
             if driver:
                 # If we encounter issues getting extra info we should silently fail rather than creating a new exception
-                try:
-                    all_logs = list(driver.get_log("browser"))
-                    scope.add_attachment(json.dumps(all_logs).encode("utf-8"), "logs.txt")
```

Comment:
Don't really have a way to support this in posthog so going to remove it for now